todoapp/app/src/main/res/layout/statistics_frag.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?><!--
-  ~ Copyright (C) 2017 The Android Open Source Project
+  ~ Copyright (C) 2015 The Android Open Source Project
```

Comment:
2015?

Reply:
Yep, it is the year it was created on. Apparently that's what you're supposed to do.